.eslintrc.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  env: {
+    commonjs: true,
+    es6: true,
+    node: true,
+  },
+  extends: [
+    'prettier',
```

Comment:
Can you ad a prettier [config file](https://github.com/prettier/eslint-plugin-prettier)